types/strategy.d.ts

```diff
@@ -33,6 +33,8 @@ declare namespace Strategy {
       dependencies: Dependencies,
       options?: HydratePayloadDependenciesOptions
     ) => Promise<void>;
+    /** Method to get vulnerabilities retrieved by arborist dependency **/
```

Comment:
Even if it's the case for now, I'm not sure we should mention `Arborist` here which is an implementation detail and is specific for the NPM strategy. What do you think?
```suggestion
    /** Method to get vulnerabilities using the current strategy **/
```

Reply:
Yes for sure, that make sens